takoyaki/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,15 +42,21 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
 
-  /**
-   * Use this to pass the autonomous command to the main {@link Robot} class.
-   *
-   * @return the command to run in autonomous
-   */
+  private void configureButtonBindings() {
+    new JoystickButton(opController, Constants.climbUp)
+      .whenPressed(new RunCommand(climb::raiseClimbStart, climb))
+      .whenReleased(climb::stopClimb, climb);
```

Comment:
whenReleased methods also require commands, not just lambdas

Reply:
```suggestion
      .whenReleased(new RunCommand(climb::stopClimb, climb));
```

this should've produced a build error, was this built before submission?

Reply:
same on line 52